.github/workflows/build.yml

```diff
@@ -0,0 +1,64 @@
+name: Build struct2tensor wheels
+
+on:
+  pull_request:
+  workflow_dispatch:
+  release:
+    types: [published]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: ["3.9", "3.10", "3.11"]
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v5
+        with:
+          python-version: ${{ matrix.python-version }}
+
+      - name: Build package
+        id: build-package
+        uses: ./.github/reusable-build
+        with:
+          python-version: ${{ matrix.python-version }}
+
+      - name: List and check wheels
+        shell: bash
+        run: |
+          # Pin transitive dependency on pkginfo until https://github.com/pypa/twine/issues/1070 is
+          # fixed
+          pip install twine pkginfo>=1.10.0
+          ${{ matrix.ls || 'ls -lh' }} dist/
+          twine check dist/*
```

Comment:
`matrix.ls` hasn't been defined in the `matrix:` field above and just `python-version:` has been defined, I'm not entirely sure what this would do?

Reply:
This is left from a build workflow that constructed cross platform wheels. If we want to have this run on windows in the future we'll need this, but since we aren't supporting that yet, I'll happily replace this.